sycl/test-e2e/Plugin/sycl-ls-unified-runtime.cpp

```diff
@@ -1,7 +1,7 @@
 // REQUIRES: TEMPORARY_DISABLED
 //  Unified Runtime will soon be changing its reporting.
 //
-// RUN: env ONEAPI_DEVICE_SELECTOR="ext_oneapi_level_zero:*" sycl-ls --verbose 2>&1 %GPU_CHECK_PLACEHOLDER
+// RUN: env ONEAPI_DEVICE_SELECTOR="ext_oneapi_level_zero:*" sycl-ls --verbose 2>&1 | FileCheck %s
```

Comment:
I think technically this changes the requirements of this test. If we are not testing for gpu then `GPU_CHECK_PLACEHOLDER` will just be empty so no FileCheck will be run. With this we would always run it, save for the fact that this test seems to be temporarily disabled. Does it matter and should we maybe require that we are currently testing for L0 too?

Reply:
I think that should be done when the test is re-enabled back. It's impossible to judge what would be the best way to adjust it now.